src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -475,7 +475,59 @@ private byte[] DecryptToken(JsonWebToken jwtToken, CryptoProviderFactory cryptoP
                     Base64UrlEncoder.DecodeBytes(jwtToken.AuthenticationTag));
         }
 
-        private string EncryptToken(string innerJwt, EncryptingCredentials encryptingCredentials, string algorithm)
+        /// <summary>
+        /// Encrypts a JWS.
+        /// </summary>
+        /// <param name="innerJwt">A 'JSON Web Token' (JWT) in JWS Compact Serialization Format.</param>
+        /// <param name="encryptingCredentials">Defines the security key and algorithm that will be used to encrypt the 'innerJwt'.</param>
+        /// <exception cref="ArgumentNullException">if 'innerJwt' is null or empty.</exception>
+        /// <exception cref="ArgumentNullException">if 'encryptingCredentials' is null.</exception>
+        /// <exception cref="ArgumentException">if both 'encryptingCredentials.CryptoProviderFactory' and 'encryptingCredentials.Key.CrypoProviderFactory' are null.</exception>
+        /// <exception cref="SecurityTokenEncryptionFailedException">if the CryptoProviderFactory being used does not support the 'encryptingCredentials.Enc' (algorithm), 'encryptingCredentials.Key' pair.</exception>
+        /// <exception cref="SecurityTokenEncryptionFailedException">if unable to create a token encryption provider for the 'encryptingCredentials.Enc' (algorithm), 'encryptingCredentials.Key' pair.</exception>
```

Comment:
nit: Consider replacing type/element references (e.g. _'encryptingCredentials.Enc'_) with the _cref_ attribute e.g. `<see cref="EncryptingCredentials.Enc"/>`.
Documentation tools use the cref attributes to automatically generate hyperlinks to the page where the type or member is documented.

Reply:
Good idea. :) I've added the 'cref' attribute everywhere I could, though I still need to figure out if there's a way to reference a property of a property (e.g. 'EncryptingCredentials.Key.CryptoProviderFactory').